src/GraphBLAS-sharp/Objects/MatrixExtensions.fs

```diff
@@ -0,0 +1,70 @@
+namespace GraphBLAS.FSharp.Objects
+
+open GraphBLAS.FSharp.Backend.Objects
+open Brahma.FSharp
+open Matrix
+
+module MatrixExtensions =
+    type ClMatrix<'a when 'a: struct> with
+        member this.ToHost(q: MailboxProcessor<_>) =
```

Comment:
Again. Why extension, not just a regular method?

Reply:
Since `GraphBLAS-sharp.Backend` doesn't know anything about fronted matrices